Dragonflight/WarriorProtection.lua

```diff
@@ -1264,7 +1263,12 @@ spec:RegisterAbilities( {
         hasteCD = true,
         gcd = "spell",
 
-        spend = function () return ( 15 + ( talent.impenetrable_wall.enabled and -5 or 0 ) + ( talent.heavy_repercussions.enabled and -18 or -15 ) ) * ( talent.unnerving_focus.enabled and 1.5 or 1) end,
+        spend = function () return 
```

Comment:
It's important to validate whether multipliers impact only the base value or the modifications from talents as well.  Have you tested the actual in-game combinations of these talents/buffs to verify?

Is it correct that Violent Outburst and Unnerving Focus increase the cost?

Reply:
I have verified that outburst enhances the rage gain on both Tclap and Shield Slam based on both outburst, unnerving focus, and both auras being up at the same time. [WCL Link](https://www.warcraftlogs.com/reports/kgNd3pFLnMwYHfrW/#type=resources&spell=101&source=7&view=events&boss=0&difficulty=0) . I am having difficulty tracking down in wow.tools or wowhead , so I'm guessing it's part of the server script portion.

Reply:
Fixed in https://github.com/Wyste/hekili/commit/101bcd4e90bfb6ffe25dd8e3fb43c7209307c34e